conformance/client/src/main/kotlin/com/connectrpc/conformance/client/ConformanceClientLoop.kt

```diff
@@ -31,16 +30,27 @@ import java.io.OutputStream
 class ConformanceClientLoop(
     private val requestUnmarshaller: (ByteArray) -> ClientCompatRequest,
     private val responseMarshaller: (ClientCompatResponse) -> ByteArray,
+    private val verbosity: Int = 0,
 ) {
     fun run(input: InputStream, output: OutputStream, client: Client) = runBlocking {
         // TODO: issue RPCs in parallel
         while (true) {
             var result: ClientCompatResponse.Result
             val req = readRequest(input) ?: return@runBlocking // end of stream
+            if (verbosity > 0) {
```

Comment:
Another option for more debugging might be to configure the okhttp logging interceptor (perhaps at higher verbosity?).

Reply:
Ack. That would likely be much more verbose than this. I haven't needed that sort of output so far, but it does seem like a potentially useful idea.